Added new fields to context output and UI (#27069)
demisto/content: Packs/PAN-OS/ReleaseNotes/1_17_4.md

#### Integrations

##### Palo Alto Networks PAN-OS

> - Added the missing data from the PAN-OS API to the context output.
> - Improved layout of the war room output display to look more like the PAN-OS UI.

Reviewer: ```suggestion
- Added additional context-output to the ***pan-os-list-rules*** command. 
```

> - Updated the Docker image to: *demisto/pan-os-python:1.0.0.58557*.

Reviewer: ```suggestion
```

u didn't update the dockerimage at all

Reviewer: ?
